README.md

```diff
@@ -217,6 +218,27 @@ export default connect(
 )(Component);
 ```
 
+### Binding actions to a outside object
+
+It's not usual, but sometimes you need to use yor actions outside from you application scope.
```

Comment:
I would change to:

`If you need to bind the actions, to an external listener, outside the application scope, here's a simple way to do it:`